examples/backupRTC/backupRTC.ino

```diff
@@ -0,0 +1,105 @@
+/*
+  backupRTC
+
+  This sketch shows how to keep  the RTC date and time with backup registers
+  on a stm32F1 MCU
+  If a VBat is present on the target board, then the content is kept when
+  the VDD power supply is off, else the content is reset
+
+  Creation 12 Dec 2017
+  by Wi6Labs
+  Modified 03 dec 2021
+  by FRASTM for STMicroelectronics
+
+  This example code is in the public domain.
+
+  https://github.com/stm32duino/STM32RTC
+*/
+
+#include <STM32RTC.h>
+
+/* Get the rtc object */
+STM32RTC& rtc = STM32RTC::getInstance();
+
+#if !defined(STM32F1xx)
+#error "This skectch is not (yet) applicable on this Target"
```

Comment:
Be more explicit to explains that it is mainly for STM32F1 as other have the feature by default.

Reply:
changed to :
`#error "Not applicable (only stm32F1xx save date in backup memory)"`